proto/config.json

```diff
@@ -0,0 +1,35 @@
+{
+  "version": "0.1.0",
+  "platform": {
+    "os": "linux",
+    "arch": "amd64"
+  },
+  "process": {
+    "terminal": true,
+    "user": {
+      "@type": "type.googleapis.com/oci.LinuxUser",
```

Comment:
Is type added by the json generation? Or is this something we have in the spec now? (I don't see it)